<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainCreateMw</class>
 <widget class="QMainWindow" name="MainCreateMw">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1141</width>
    <height>835</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Auto Create 4.5</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>timgCADXKVUY.ico</normaloff>timgCADXKVUY.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonStart">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>220</y>
      <width>93</width>
      <height>49</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Agency FB&quot;;</string>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStop">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>220</y>
      <width>93</width>
      <height>49</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Agency FB&quot;;</string>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClear">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>220</y>
      <width>93</width>
      <height>49</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Agency FB&quot;;</string>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>1101</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(224, 255, 224);
font: 10pt &quot;Arial&quot;;</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>41</y>
      <width>1091</width>
      <height>134</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Testcase Path:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditTSPath">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(170, 170, 255);
font: 10pt &quot;Agency FB&quot;;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;A2L Path:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdita2lPath">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(170, 170, 255);
font: 10pt &quot;Agency FB&quot;;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Model Map:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditModelmap">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(170, 170, 255);
font: 10pt &quot;Agency FB&quot;;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;User Map:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditUsermap">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(170, 170, 255);
font: 10pt &quot;Agency FB&quot;;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Create Type:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditCreatetype">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(170, 170, 255);
font: 10pt &quot;Agency FB&quot;;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;HIL Type:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditHILType">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(170, 170, 255);
font: 10pt &quot;Agency FB&quot;;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1111</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(255, 255, 0);</string>
    </property>
    <property name="title">
     <string>Information</string>
    </property>
   </widget>
   <zorder>pushButtonClear</zorder>
   <zorder>textBrowser</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>pushButtonStart</zorder>
   <zorder>pushButtonStop</zorder>
   <zorder>groupBox</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1141</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNew_Style">
    <property name="title">
     <string>Create Pkg</string>
    </property>
    <addaction name="ActNewOpen"/>
   </widget>
   <widget class="QMenu" name="menuOld_Style">
    <property name="title">
     <string>Create DB</string>
    </property>
    <addaction name="ActCreateDB"/>
   </widget>
   <addaction name="menuNew_Style"/>
   <addaction name="menuOld_Style"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="ActNewOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="ActCreateDB">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="xiaoniao.qrc"/>
 </resources>
 <connections/>
</ui>
